azure-pipelines/start-emulator.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# Install AVD files
+# TODO: Do not hardcode Android SDK versions
+echo "y" | $ANDROID_HOME/tools/bin/sdkmanager --install 'system-images;android-28;google_apis;x86'
+
+# Create emulator
+echo "no" | $ANDROID_HOME/tools/bin/avdmanager create avd -n testemulator -k 'system-images;android-28;google_apis;x86' --force
+
+echo $ANDROID_HOME/emulator/emulator -list-avds
+
+echo "Starting emulator"
+
+# Start emulator in background
+nohup $ANDROID_HOME/emulator/emulator -avd testemulator -no-snapshot > /dev/null 2>&1 &
+$ANDROID_HOME/platform-tools/adb wait-for-device shell 'while [[ -z $(getprop sys.boot_completed | tr -d '\r') ]]; do sleep 1; done; input keyevent 82'
```

Comment:
this condition is not always what is needed. Consider checking https://github.com/appium/java-client/blob/392413f9d223da119000d9850b4237a07ea7a530/azure-pipelines.yml#L28